style.css

```diff
@@ -1503,3 +1507,435 @@
   font-size: 23px;
   margin-left: 18px;
 }
+
+@media only screen and (min-width: 768px) {
```

Comment:
You have done great implementing the desktop version of the portfolio. Kindly ensure that the mobile version is responsive until the desktop screen size (768px or larger).

Your design at 558px:
![image](https://user-images.githubusercontent.com/20554583/215263109-48b343a5-47c6-4265-95c1-c526875cabc2.png)

Reply:
Done.